examples/typescript/__tests__/sum-test.ts

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 
+import sum from '../sum';
+
 it('adds 1 + 2 to equal 3 in TScript', () => {
-  const sum = require('../sum.ts').default;
```

Comment:
should be kept as is I believe - it's explicitly loading `.ts`

Reply:
Yeah, OK. I think the reason for my change was to hopefully prevent people from doing this since it results in `any`. But I guess this example is showing how you can mix and match.

Reply:
can add a comment above that it should be discouraged